From bd741faa4ecd081b4898a259b6b124e22644eca0 Mon Sep 17 00:00:00 2001
From: Rob Landley <rob@landley.net>
Date: Fri, 15 Mar 2024 17:52:44 -0500
Subject: [PATCH 4/4] Fix first line summary for xzcat.

---
 toys/pending/xzcat.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/toys/pending/xzcat.c b/toys/pending/xzcat.c
index 7f85d9e0..ec0e2c3d 100644
--- a/toys/pending/xzcat.c
+++ b/toys/pending/xzcat.c
@@ -1,5 +1,4 @@
-/*
- * Simple XZ decoder command line tool
+/* xzcat.c - Simple XZ decoder command line tool
  *
  * Author: Lasse Collin <lasse.collin@tukaani.org>
  *
-- 
2.44.0

